portals/knowledge-management/src/portal.js

```diff
@@ -70,6 +70,11 @@ customElements.define(
                         name: 'view-category',
                         component: ViewCategory,
                     },
+                    {
+                        path: baseUrl + '/categories/:parentCategorySlug/:categorySlug',
```

Comment:
When I navigate to a SubCategory page I am currently not seeing this URL structure, even in incognito mode. Are you experiencing the same?